cron.go

```diff
@@ -43,31 +48,10 @@ type Job struct {
 	Name string
 	// Cron-formatted rhythm (ie. 0,10,30 1-5 0 * * *)
 	Rhythm string
-	// Routine method
-	Func func(context.Context) error
-
-	Repeats  int32
-	DueTime  string
-	TTL      string
-	Data     *anypb.Any
-	Metadata map[string]string
-}
-
-func (j Job) Run(ctx context.Context) error {
-	return j.Func(ctx)
-}
-
-var (
-	nonAlphaNumerical = regexp.MustCompile("[^a-z0-9_]")
-)
-
-func (j Job) canonicalName() string {
-	return strcase.ToSnake(
-		nonAlphaNumerical.ReplaceAllString(
-			strings.ToLower(j.Name),
-			"_",
-		),
-	)
+	// The type of trigger
+	TriggerType string
+	// The payload containg all the information for the trigger
+	TriggerPayload []byte
```

Comment:
I will convert this to Any again and use protobuf internally to save the state.

Reply:
Were you going to do this change in this PR or a follow up? I guess it wont hurt to merge since we aren't referencing the commit in dist scheduler yet, but still feel we should update it first.